docs/sphinx/users/micromanager/index.txt

```diff
@@ -0,0 +1,39 @@
+Micro-Manager
+=============
+
+- Micromanager offers the functionality for saving the acquired images in tiff/ome-tiff format. 
+- Based on the mode of saving and the configuration settings, the acquired image can be saved 
+  with/without a companion file (metadata.txt).
```

Comment:
I would also make these first two points just two sentences of text, there isn't really any need for them to be a bullet list.